### DISCUSSION TITLE
Hassio non riparte

### DISCUSSION THREAD

**Discussion by maxdanluca** [indomus/forum#2638] [GENERALE - Discussioni generiche]
dopo aver aggiunto un Kingston A400 SSD Unità a stato solido al posto della scheda SD il mio Hassio dopo un black out non riparte in automatico ma devo spegnere e riaccendere il raspberry P3. cosa devo fare? Grazie e abbiate pietà di un vecchio di 67 anni

**Comment by essegi65** [comment]
Hai provato ad attaccare un monitor al RPI 3 e vedere se ti appaiono errori in fase di avvio?
